decidim-comments/app/models/decidim/comments/comment.rb

```diff
@@ -63,6 +63,11 @@ def reported_content_url
         ResourceLocatorPresenter.new(root_commentable).url(anchor: "comment_#{id}")
       end
 
+      # Public: Returns the comment message ready to display (it is expected to include HTML)
+      def formatted_body
+        Decidim::ContentProcessor.render(sanitized_body)
```

Comment:
Shouldn't we memoize this?

`@formatted_body ||= Decidim::ContentProcessor.render(sanitized_body)`